person.rb

```diff
@@ -1,10 +1,12 @@
-class Person
-  attr_accessor :name, :age
+require './nameable'
+
+class Person < Nameable
+  attr_accessor :age
   attr_reader :id
 
-  def initialize(age, parent_permission: true, name: 'Unknown')
+  def initialize(age, parent_permission: true)
+    super()
     @id = rand(1..1000)
-    @name = name
     @age = age
     @parent_permission = parent_permission
   end
```

Comment:
- [ ] Kindly pay attention to the details. In your change you removed `name` from the parameters, which means we can't even create a person object at all. You also have not added the line `@name = name`. These cause errors:

  ![image](https://user-images.githubusercontent.com/54848363/231441506-54bec03c-be6f-4dab-b047-18254ea14f4a.png)

  Kindly ensure your code matches this:
  
  ```
  def initialize(age, parent_permission: true, name: 'Unknown')
    super()
    @id = rand(1..1000)
    @age = age
    @name = name
    @parent_permission = parent_permission
  end
  ```

Reply:
Thank you, fixed.